CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/RunPane.java

```diff
@@ -186,6 +189,19 @@ public RunPane(final ImportController importController) {
 
         attributePane.addRow(0, sourceVertexScrollPane, destinationVertexScrollPane, transactionScrollPane);
 
+        attributePane.setOnKeyPressed(event -> {
```

Comment:
In the interest of good UX, I would strongly recommend having a visual element to this filter. Having it just hide attributes as you type with no visual feedback is begging for erroneous bug reports.

Reply:
I've added a blurb to the 'help text' on that middle section and the responsiveness of it changing is fairly clear that you caused something to change when you start typing.  I did have it as a text edit button under Options, but that sucked

Reply:
What about something like, when you start typing, a text input pops up to show that you're doing something?

Reply:
Have added the text field in.